components/Programs/Dos.tsx

```diff
@@ -0,0 +1,77 @@
+import styles from '@/styles/Programs/Dos.module.scss';
```

Comment:
Logic of import order (space between "types"):
- Files (png, svg, json, etc.)
- Styles
- Types modules then exports (alphabetical)
- Modules then exports (alphabetical)

NOTE: Perhaps add a style guide?